src/components/review/EssentialStudyMaterial.astro

```diff
@@ -2,38 +2,38 @@
 const list1 = [
   {
     name: "Common operational steps of Core Review Process",
-    link: "",
-    code: "dummylink",
+    link: "https://youtu.be/n5CRJRqkAoc",
+    code: "https://github.com/Bitshala/BitcoinCore-PR-Review-Club/blob/main/test-pr.sh",
   },
   {
     name: "Bitcoin Core Codebase Introductory Tour",
-    link: "",
+    link: "https://www.youtube.com/watch?v=MbinzItqsXI",
   },
 ];
 const list2 = [
   {
     name: "Contributing to Bitcoin Core",
-    link: "",
+    link: "https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md",
   },
   {
     name: "Developer Notes",
-    link: "",
+    link: "https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md",
   },
   {
     name: "Introduction to Bitcoin Core Development",
-    link: "",
+    link: "https://bitcointechtalk.com/a-gentle-introduction-to-bitcoin-core-development-fdc95eaee6b8",
   },
   {
     name: "Contributing to Bitcoin Core, a personal account",
-    link: "",
+    link: "https://bitcointechtalk.com/contributing-to-bitcoin-core-a-personal-account-35f3a594340b",
```

Comment:
New link

https://johnnewbery.com/contributing-to-bitcoin-core-a-personal-account/